src/components/proposal/ProposalAdmin.tsx

```diff
@@ -1,10 +1,11 @@
-import { administrationProposalValidationSchema } from '@esss-swap/duo-validation/lib/Proposal';
+// import { administrationProposalValidationSchema } from '@esss-swap/duo-validation/lib/Proposal';
```

Comment:
I think we should not leave commented code, as SVC keeps the history

Reply:
After the validation package is upgraded I'll restore them.